src/code/z_player_lib.c

```diff
@@ -44,197 +45,224 @@ u8 gPlayerModelTypes[][5] = {
+    gLinkChildRightHandHoldingSlingshotFarDL,
 };
 
-Gfx* D_80125EB8[] = {
-    0x06024698,
+Gfx* sFairyOcarinaDLs[] = {
+    gLinkAdultRightHandHoldingOotNearDL,
     gLinkChildRightHandHoldingFairyOcarinaNearDL,
-    0x06028F58,
+    gLinkAdultRightHandHoldingOotFarDL,
     gLinkChildRightHandHoldingFairyOcarinaFarDL,
 };
 
-Gfx* D_80125EC8[] = {
-    0x06024698,
+Gfx* sOotDLs[] = {
+    gLinkAdultRightHandHoldingOotNearDL,
     gLinkChildRightHandAndOOTNearDL,
-    0x06028F58,
+    gLinkAdultRightHandHoldingOotFarDL,
     gLinkChildRightHandHoldingOOTFarDL,
 };
 
-Gfx* D_80125ED8[] = {
-    0x06024D70,
+//! @bug The 'far' display list is never used
+Gfx* sHookshotDLs[] = {
+    gLinkAdultRightHandHoldingHookshotNearDL,
     gLinkChildRightHandNearDL,
-    0x06024D70,
+    gLinkAdultRightHandHoldingHookshotNearDL,
     gLinkChildRightHandFarDL,
 };
 
-Gfx* D_80125EE8[] = {
-    0x060233E0,
+Gfx* sHammerDLs[] = {
+    gLinkAdultLeftHandHoldingHammerNearDL,
     gLinkChildLeftHandNearDL,
-    0x06028288,
+    gLinkAdultLeftHandHoldingHammerFarDL,
     gLinkChildLeftHandFarDL,
 };
 
-Gfx* D_80125EF8[] = {
-    0x06021AA8,
+Gfx* gPlayerBoomerangDLs[] = {
+    gLinkAdultLeftHandNearDL,
     gLinkChildLeftFistAndBoomerangNearDL,
-    0x06026C58,
+    gLinkAdultLeftHandFarDL,
     gLinkChildLeftFistAndBoomerangFarDL,
 };
 
 Gfx* D_80125F08[] = {
-    0x06024B58,
+    gLinkAdultLeftHandOutNearDL,
     gLinkChildLeftHandUpNearDL,
-    0x06024B58,
+    gLinkAdultLeftHandOutNearDL,
     gLinkChildLeftHandUpNearDL,
 };
 
-Gfx* D_80125F18[] = {
-    0x06029918,
+Gfx* sArmOutDLs[] = {
+    gLinkAdultRightArmOutNearDL,
     NULL,
 };
 
-Gfx* D_80125F20[] = {
-    0x06029C20,
+Gfx* sHandOutDLs[] = {
+    gLinkAdultRightHandOutNearDL,
     NULL,
 };
 
-Gfx* D_80125F28[] = {
-    0x06036E58,
-    gLinkChildLeftShoulderNearDL,
+Gfx* sRightShoulderNearDLs[] = {
+    gLinkAdultRightShoulderNearDL,
+    gLinkChildRightShoulderNearDL,
 };
 
 Gfx* D_80125F30[] = {
-    0x06029FA0,
+    gLinkAdultLeftArmOutNearDL,
     NULL,
 };
 
-Gfx* D_80125F38[] = {
-    0x0602A248,
+Gfx* sHoldingFirstPersonWeaponDLs[] = {
+    gLinkAdultRightHandHoldingBowFirstPersonDL,
     gLinkChildRightArmStretchedSlingshotDL,
 };
 
 // Indexed by model types (left hand, right hand, sheath or waist)
 Gfx** sPlayerDListGroups[] = {
-    D_80125E08, D_80125E18, D_80125E38, D_80125E28, D_80125DE8, D_80125EE8, D_80125EF8,
-    D_80125F08, D_80125E48, D_80125E58, D_80125CE8, D_80125E68, D_80125EA8, D_80125EB8,
-    D_80125EC8, D_80125ED8, D_80125E78, D_80125E88, D_80125D28, D_80125D88, D_80125E98,
+    gPlayerLeftHandDLs, D_80125E18,    D_80125E38,           D_80125E28,          D_80125DE8, sHammerDLs, gPlayerBoomerangDLs,
+    D_80125F08,   sRightHandDLs, sRightHandClosed2DLs, sRightHandClosedDLs, sHandHoldingFirstPersonWeaponDLs, D_80125EA8, sFairyOcarinaDLs,
+    sOotDLs,      sHookshotDLs,  sSwordAndSheathDLs,   sSheathDLs,          D_80125D28, D_80125D88, sWaistDLs,
```

Comment:
I feel like we should probably let whoever does model group documentation name these arrays. As is, I dont think I agree with some of the names here. For example the hand dlist arrays are probably going to be more appropriately named based on the model group they are indexed by/belong to rather than naming it after the fact that his hand is closed. And the bow/slingshot one being named first person weapon, when there are other first person weapons is unfortunate

Basically, I understand that it feels all over the place, and thats cause the model group system kinda is, but I think they should be documented together. I would personally just revert all of these names for now